vitrina/structure/services.py

```diff
@@ -1298,6 +1328,12 @@ def _get_access(acess: int) -> str:
     return ""
 
 
+def _get_title(obj: Status | Visibility) -> str:
+    if obj is None:
+        return ""
+    return obj.title
```

Comment:
Might as well make it a one liner:

```suggestion
    return "" if obj is None else obj.title
```